tests_integration/test_cortex.py

```
@@ -0,0 +1,21 @@
+import pytest
+
+
+@pytest.mark.integration
+@pytest.mark.skip  # TODO: when we have Cortex activated on test account, unskip this and setup part in sql script
+def test_cortex_search(runner):
+    result = runner.invoke_with_connection_json(
+        [
+            "cortex",
+            "search",
+            "parrot",
+            "--service",
+            "test_service",
+            "--limit",
+            "1",
+            "--columns",
+        ]
```

How does it work? You haven't provided any columns, this should throw an error?